### PR TITLE
Update attack_defense.json

### PR DESCRIPTION
**Pull request by parrotassassin15**

### CODE REVIEW

File: data/ctf_platforms/attack_defense.json [hunk 1]
```diff
@@ -45,6 +45,13 @@
       "language": "Python",
       "price": "Free"
     },
+    {
+      "name": "Parrot CTFs",
+      "source": "https://github.com/Parrot-CTFs",
```
Comment (noraj) [inline data/ctf_platforms/attack_defense.json:50]: Could you add the website too? https://parrot-ctfs.com/

File: data/ctf_platforms/attack_defense.json [hunk 2]
```diff
@@ -45,6 +45,13 @@
       "language": "Python",
       "price": "Free"
     },
+    {
+      "name": "Parrot CTFs",
```
Comment (noraj) [inline data/ctf_platforms/attack_defense.json:49]: Could you sort it alphabetically please?

**Review by noraj — CHANGES_REQUESTED** [review]

File: data/ctf_platforms/attack_defense.json [hunk 3]
```diff
@@ -45,6 +45,13 @@
...
+      "source": "https://github.com/Parrot-CTFs",
+      "description": "Attack-defence CTF framework",
```
Comment (noraj) [inline data/ctf_platforms/attack_defense.json:51]: I don't see any mention to Attack-defence anywhere, it seems to be a jeopardy platform, could you move it to this category and update the description?

**Review by noraj — CHANGES_REQUESTED** [review]

File: data/ctf_platforms/attack_defense.json (hunk 2 above)
Comment (parrotassassin15) [reply data/ctf_platforms/attack_defense.json:49]: yes I can. I am sorry I was not online lemme fix all this now.

**Pull request closed without merging**